network/src/commonMain/kotlin/com/wire/kalium/network/api/v0/authenticated/ConversationApiV0.kt

```diff
@@ -260,38 +260,38 @@ internal open class ConversationApiV0 internal constructor(
             }
         }
 
+    @Suppress("MagicNumber")
+    override suspend fun fetchMlsOneToOneConversation(userId: UserId): NetworkResponse<ConversationResponse> =
+        getApiNotSupportError(::fetchMlsOneToOneConversation.name, 4)
+
+    @Suppress("MagicNumber")
     override suspend fun fetchSubconversationDetails(
         conversationId: ConversationId,
         subconversationId: SubconversationId
     ): NetworkResponse<SubconversationResponse> =
-        NetworkResponse.Error(
-            APINotSupported("MLS: fetchSubconversationDetails api is only available on API V3")
-        )
+        getApiNotSupportError(::fetchSubconversationDetails.name, 4)
 
+    @Suppress("MagicNumber")
     override suspend fun deleteSubconversation(
         conversationId: ConversationId,
         subconversationId: SubconversationId,
         deleteRequest: SubconversationDeleteRequest
     ): NetworkResponse<Unit> =
-        NetworkResponse.Error(
-            APINotSupported("MLS: deleteSubconversation api is only available on API V3")
-        )
+        getApiNotSupportError(::deleteSubconversation.name, 4)
 
+    @Suppress("MagicNumber")
     override suspend fun fetchSubconversationGroupInfo(
         conversationId: ConversationId,
         subconversationId: SubconversationId
     ): NetworkResponse<ByteArray> =
-        NetworkResponse.Error(
-            APINotSupported("MLS: fetchSubconversationGroupInfo api is only available on API V3")
-        )
+        getApiNotSupportError(::fetchSubconversationGroupInfo.name, 4)
 
+    @Suppress("MagicNumber")
     override suspend fun leaveSubconversation(
         conversationId: ConversationId,
         subconversationId: SubconversationId
     ): NetworkResponse<Unit> =
-        NetworkResponse.Error(
-            APINotSupported("MLS: leaveSubconversation api is only available on API V3")
-        )
+        getApiNotSupportError(::leaveSubconversation.name, 4)
```

Comment:
Nice `getApiNotSupportError` function. Maybe we can extract the 4 to const something like, `MIN_VERSION_SUPPORT` or something like that, to avoid the `@Suppress("MagicNumber")`

Reply:
Done